test/player.spec.js

```diff
@@ -0,0 +1,29 @@
+import { Player } from '../src/app/Player';
+
+describe("Player's logic", () => {
+  const player = new Player();
+  let question;
+
+  it('When player is asked, call onQuestionAsked function with question as parameter', () => {
+    const onQuestionAsked = jest.fn();
+
+    player.askQuestion(question, onQuestionAsked);
+
+    expect(onQuestionAsked).toBeCalledWith(question);
+  });
+
+  it('When player is asked and onQuesionAsked function is not used, check if there is no error', () => {
```

Comment:
Jest ok tylko tutaj dalbym function is undefined (nie not used).
i w ogole tak:
`expect(() => player.askQuestion(question)).not.toThrow();`
Nie musisz deklarowac, ani przekazywac parametru, wtedy wlasnie jest undefined :) 
Do drugiej funkcji answer to samo i jest OK.

Reply:
done